### ISSUE TITLE
关于训练DA-CLIP

### ISSUE THREAD

**Issue by c-pupil** [Algolzw/daclip-uir#91]
Dear author:
  您好，首先感谢您出色的工作，其次我想问您一下我想按照您的方式训DA-CLIP，在我自己的训练集重头训练，我的训练集800张，用单张3090，按照下面的超参设置，每一轮大约几秒钟，显存大约15G，我想问这合理么（训练速度和占内存很令我感到意外）
`python -m training.main \
    --save-frequency 50 \
    --zeroshot-frequency 1 \
    --report-to tensorboard \
    --train-data="daclip_train.csv"  \
    --val-data="daclip_val.csv"  \
    --csv-img-key filepath \
    --csv-caption-key title \
    --warmup 100 \
    --batch-size=400 \
    --lr=2e-5 \
    --wd=0.05 \
    --epochs=500 \
    --workers=8 \
    --model daclip_ViT-B-32 \
    --name "daclip_ViT-B-32_b784x1_lr2e-5_e50" \
    --pretrained "laion2b_s34b_b79k" \
    --da`

**Comment by Algolzw** [comment]
应该是合理的，因为训练集太小而batch size太大。你可以做一些数据增强来避免过拟合。
